client/src/feedbacker-components/dashboard-link/dashboard-link.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import configureMockStore from 'redux-mock-store'
+import { Provider } from 'react-redux'
+import DashboardLink from './dashboard-link'
+
+let mockStore
+let middlewares
+
+beforeEach(() => {
+  middlewares = []
+  mockStore = configureMockStore(middlewares)
+  global.window.location =  { href: 'site.localhost:8080' }
+})
+
+test('should have back to dashboard text on button', () => {
+  const store = mockStore()
+  const view = mount(
+    <Provider store={store}>
+      <DashboardLink />
+    </Provider>
+  )
+  const wrapper = view.find('.dashboard-button')
+  expect(wrapper.text()).toBe('Back to dashboard')
+})
+
+test('link should be correct', () => {
+  global.window = Object.create(window)
+  const url = 'https://new-feature-4cdfa.feedbacker.site'
+  Object.defineProperty(window, 'location', {
+    value: {
+      origin: url,
+    },
+  })
+  const store = mockStore()
+  const view = mount(
+    <Provider store={store}>
+      <DashboardLink />
+    </Provider>
+  )
+  view.setProps({ href: `//${window.location.origin.match(/([^.]*).(.*)/m)[2]}` })
```

Comment:
why not use baseDomain() now duplicated code, what if function code changes?

Reply:
Could probably use `expect(wrapper.prop('href')).toBe(...)`

Reply:
As this will use the href generated by `baseDomain()`